msctl

```diff
@@ -1825,6 +1956,10 @@ elif [ -r "$HOME/mscs.defaults" ]; then
 elif [ -r "$HOME/.config/mscs/mscs.defaults" ]; then
   MSCS_DEFAULTS="$HOME/.config/mscs/mscs.defaults"
 fi
+if [ ! -r "$MSCS_DEFAULTS" ]; then
```

Comment:
This should be `-s` (True if file exists and has a size greater than zero) rather than `-r` (True if file exists and is readable).